component styles/Alert.css

```diff
@@ -0,0 +1,49 @@
+.alert-container{
+    width: 90vw;
+    margin: 0 auto;
+    max-width: 40rem; 
```

Comment:
why are you using width and max-width here

Reply:
what this does is, it keeps width at 90vw always and keeps it centered because of margin: 0 auto. max-width - will not allow the width to go beyond some value. This will help when we shrink the the browser window till breakpoint.

Reply:
okay